ci/test.yml

```diff
@@ -46,33 +46,56 @@
       - trezor-user-env-debugging.log
       - tenv-emulator-bridge-debugging.log
       - trezor-user-env-version.txt
+
+.suite web base:
+  extends: .e2e web
   parallel:
     matrix:
       - TEST_GROUP: ['@group:suite', '@group:onboarding', '@group:device-management', '@group:settings', '@group:metadata']
         CONTAINERS: 'trezor-user-env-unix'
       - TEST_GROUP: '@group:wallet'
         CONTAINERS: 'trezor-user-env-unix bitcoin-regtest'
 
-suite web:
+# fragile group is dependent on external powers which means it might fail when some external resource
+# is modified. We still believe it might be worth having to fix such broken test time to time.
+.suite web fragile base:
   extends: .e2e web
+  allow_failure: true # most importantly - when a test in this category fails, it does not make entire pipeline fail
+  parallel:
+    matrix:
+      - TEST_GROUP: '@group:fragile'
+        CONTAINERS: 'trezor-user-env-unix'
+
+suite web:
+  extends: .suite web base
 
 suite web snapshots:
-  extends: .e2e web
+  extends: .suite web base
+  when: manual
+  variables:
+    CYPRESS_SNAPSHOT: 1
+    CYPRESS_updateSnapshots: 1
+
+suite web qa:
```

Comment:
is the job name intentionally with `qa` instead of `fragile`?
![image](https://user-images.githubusercontent.com/3729633/159297100-7726107e-ee93-4ef7-b270-4a1a3ba8c8d7.png)